package.json

```diff
@@ -24,7 +24,7 @@
   "dependencies": {
     "electron": "1.4.1",
     "electron-edge": "6.5.4",
-    "gpii-windows": "git://github.com/danailbd/windows.git#hst-2017",
+    "gpii-windows": "git://github.com/danailbd/windows.git",
     "infusion": "3.0.0-dev.20170830T182157Z.392b2f8",
     "node-jqunit": "1.1.7",
```

Comment:
Please re-merge up with trunk of gpii-app to take in recent dependencies plus renamed repository gpii-universal